apis/python/src/tiledbsoma/dataframe.py

```diff
@@ -67,105 +50,47 @@ def create_legacy(
 
         :param platform_config: Platform-specific options used to create this DataFrame, provided via "tiledb"->"create" nested keys
         """
-        util.check_type("schema", schema, (pa.Schema,))
-
-        schema = _validate_schema(schema, index_column_names)
-        self._create_empty(
+        context = context or SOMATileDBContext()
+        schema = _canonicalize_schema(schema, index_column_names)
+        tdb_schema = _build_tiledb_schema(
             schema,
             index_column_names,
             TileDBCreateOptions.from_platform_config(platform_config),
+            context,
+        )
+        handle = cls._create_internal(uri, tdb_schema, context)
+        is_sparse = tdb_schema.sparse
+        return cls(
+            uri,
+            "w",
+            handle,
+            context,
+            _is_sparse=is_sparse,
```

Comment:
given that all SOMA DataFrames are sparse (as of Nov 2022), this attribute seems like dead code.  Can we remove it?  I couldn't find anything that actually uses the value.....